specs/_features/eip7594/fork-choice.md

```diff
@@ -0,0 +1,122 @@
+# EIP-7594 -- Fork Choice
+
+## Table of contents
+<!-- TOC -->
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Introduction](#introduction)
+  - [Helpers](#helpers)
+    - [Modified `is_data_available`](#modified-is_data_available)
+    - [Modified `get_head`](#modified-get_head)
+- [Updated fork-choice handlers](#updated-fork-choice-handlers)
+  - [Modified `on_block`](#modified-on_block)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+<!-- /TOC -->
+
+## Introduction
+
+This is the modification of the fork choice accompanying EIP-7594.
+
+### Helpers
```

Comment:
Just a nit, I think we should rename this to "Helper functions" for consistency with other specs, eg:

<img width="541" alt="image" src="https://github.com/user-attachments/assets/83ce3b20-73c6-4c89-a43c-586785df9306">

Reply:
I left this one as is for now, because "Helpers" is what is used in the fork-choice specs, both for phase0 and Deneb, so there's anyway an inconsistency. Perhaps we can just change it everywhere